spec/scrabble_spec.rb

```diff
@@ -18,6 +18,11 @@
       expect(Scrabble::Scrabble.score("quiz")).to eq 22
       expect(Scrabble::Scrabble.score("it")).to eq 2
       expect(Scrabble::Scrabble.score("HanDLeR")).to eq 61
+    end
```

Comment:
It's a great idea to test that mixed-case words behave normally (but it wouldn't be a bad idea to pull that out into a separate test, to be explicit that that's what you're testing.)